lib/Cake/Core/App.php

```diff
@@ -889,6 +889,20 @@ protected static function _packageFormat() {
 	}
 
 /**
+ * Increases the PHP "memory_limit" ini setting by the specified amount
+ * in megabytes  
+ *  
+ * @param string $additionalMb Number in megabytes
+ * @return void
+ */
+	public static function increasePhpMemoryLimit($additionalMb) {
+		$limit = ini_get("memory_limit");
+		if (is_string($limit) && substr($limit, -1) == 'M') {
```

Comment:
so if I allocate 1G of memory, this won't do anything - thats a weird and unintended behavior ? :)